doc/vim-go.txt

```diff
@@ -1485,9 +1485,9 @@ Specifies whether `gocode` should use a different socket type. By default
                                                   *'g:go_template_autocreate'*
 
 When a new Go file is created, vim-go automatically fills the buffer content
-with a Go code template. By default the template under
-`templates/hello_world.go` is used. This can be changed with the
-|'g:go_template_file'| setting.
+with a Go code template. By default, the templates under `templates` are used.
```

Comment:
I think `under templates folder are used ...` is better. People might not know that we have a `templates` folder.